# To create the joint angles set for storing complete robot trajectory in joint config 

gen_utilities/KukaJointAngles[] joint_angles_set
gen_utilities/IdxGroup[] group_idx 